ecosystem/typescript/sdk/src/transaction_builder/bcs/helper.ts

```diff
@@ -40,8 +40,50 @@ export function bcsSerializeUint64(value: AnyNumber): Bytes {
   return serializer.getBytes();
 }
 
+export function bcsSerializeU8(value: Uint8): Bytes {
+  const serializer = new Serializer();
+  serializer.serializeU8(value);
+  return serializer.getBytes();
+}
+
+export function bcsSerializeU16(value: Uint16): Bytes {
+  const serializer = new Serializer();
+  serializer.serializeU16(value);
+  return serializer.getBytes();
+}
+
+export function bcsSerializeU32(value: Uint32): Bytes {
+  const serializer = new Serializer();
+  serializer.serializeU32(value);
+  return serializer.getBytes();
+}
+
+export function bcsSerializeU128(value: AnyNumber): Bytes {
+  const serializer = new Serializer();
+  serializer.serializeU128(value);
+  return serializer.getBytes();
+}
+
+export function bcsSerializeBool(value: boolean): Bytes {
+  const serializer = new Serializer();
+  serializer.serializeBool(value);
+  return serializer.getBytes();
+}
+
 export function bcsSerializeStr(value: string): Bytes {
   const serializer = new Serializer();
   serializer.serializeStr(value);
   return serializer.getBytes();
 }
+
+export function bcsSerializeBytes(value: Bytes): Bytes {
+  const serializer = new Serializer();
+  serializer.serializeBytes(value);
+  return serializer.getBytes();
+}
+
+export function bcsSerializeFixedBytes(value: Bytes): Bytes {
+  const serializer = new Serializer();
+  serializer.serializeFixedBytes(value);
+  return serializer.getBytes();
+}
```

Comment:
What's the difference?

Reply:
bcsSerializeBytes would serialize a length and then the bytes sequence.
 bcsSerializeFixedBytes would serialize the bytes sequence without a length ahead. Because fixed bytes's length is known. e.g. AccountAddress's length is 32. BCS doesn't need to serialize the length.